### DISCUSSION TITLE
Storybook addon - Use the new Framework API

### DISCUSSION THREAD

**Discussion by jonniebigodes** [urql-graphql/urql#2943] [Ideas]
Hi folks 👋 !
I'm one of the Storybook maintainers. I focus primarily on documentation and community outreach. I'm reaching out to get some information regarding your future plans for the Storybook addon you've developed and if you're considering supporting the upcoming Storybook 7.0 release. To help you with this, we've prepared an abridged guide for the required changes [here](https://chromatic-ui.notion.site/Addon-migration-for-Storybook-7-002f2f627c1d470298ca94894ab8a040), and if you have any questions or issues, please reach out to us in the #prerelease channel in our [Discord Server](https://discord.gg/storybook). 

Looking forward to hearing from you.

Hope you have a fantastic day.

Stay safe

**Comment by kitten** [comment]
Hiya 👋 
Thanks for the heads up! I'll have a look at what the change entails and hope it'll give us some new ideas and make the addon a little more usable and practical!
Cheers!

**Reply by jonniebigodes** [reply]
Thanks for following up with me @kitten, and no need to thank whatsoever. It was my pleasure. If you have any issues/ questions, reach out to me directly (same username) on our Discord server or here so that we can polish the documentation accordingly.

Looking forward to seeing the changes.

**Comment by JReinhold** [comment]
Hi @kitten, Storybook v7.0.0 is out! It's only on the `next` tag for now, but it's still targetable with the `^7.0.0` semver range, so you should be able to develop against it with no dependency issues.

If you hit any issues with v7, we've written a migration guide for addon authors [here](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#specific-instructions-for-addon-creators) and [here](https://chromatic-ui.notion.site/Addon-migration-for-Storybook-7-002f2f627c1d470298ca94894ab8a040).

You're also free to reach out to us in the #prerelease channel on our [Discord Server](https://discord.gg/storybook) and we'll happily help you out!

cc @ndelangen

**Reply by JReinhold** [reply]
Sorry, I didn't realize you had already deprecated it [here](https://github.com/urql-graphql/urql/pull/3067). Please ignore me then.
